docs/ENVIRONMENT.md

```diff
@@ -279,3 +279,8 @@ systemd-sysext:
   specify API file systems such as `/proc/` or `/sys/` here, or hierarchies
   that have them as submounts. In particular, do not specify the root directory
   `/` here.
+
+systemd-tmpfiles:
+
+* `$SYSTEMD_TMPFILES_FORCE_SUBVOL=1` - if set, tmpfiles will not check if the
+   system is installed into a btrfs subvolume when evaluating the v/q/Q line types
```

Comment:
key here is that the env var *overrides* the automatic check in all ways.

i.e. this should say:

if unset, v/q/Q lines will create subvolumes only if the OS itself is installed into a subvolume. If this variable is 1 (or any other true value) these lines will always create subvolumes (as long as the backing file system supports them), if it is 0 (or any false value) these lines will always create directories (even if the backing file system supports subvolumes).

or something like that